### ISSUE TITLE
Cross-entropy cost for SoftmaxOutput as evaluation metric

### ISSUE THREAD

**Issue by hesseltuinhof** [apache/incubator-mxnet#7290]
I am wondering if there really is no implementation of the cross-entropy cost function as a evaluation metric when doing multi-class classification? For the binary case (i.e. logistic regression) it is provided.

I do understand that for training it is not necessary to calculate the cross-entropy but still it is a valuable evaluation metric to monitor the training process of a multi-class classifier.

I did some extensive googling, but couldn't find any implementation of the cross-entropy cost as a evaluation metric in Python (as mentioned above, only the binary case is implemented). So is there really no implementation of it available?

**Comment by hesseltuinhof** [comment]
The definition of the cross-entropy in the documentation mislead me to believe that the `mxnet.metric.CrossEntropy()` was only implemented for logistic regression. However I was wrong! It is of course also working for multi-class....
